From 6316090f160bf91529240fba0b2b000fa1ad3dfe Mon Sep 17 00:00:00 2001
From: Sai Surya Teja <sssteja@habana.ai>
Date: Fri, 24 Sep 2021 12:28:12 +0300
Subject: [PATCH] [SW-56687] ppboot: size check for ppboot binary

Enabled size check for ppboot binary in gaudi2 arc
Gaudi2 arc Ppboot build will fail if binary size is greater than 16KB

We cannot allow large foot print of ppboot as it will increase timing
for arcboot to copy from flash to SRAM and by that will japrodize
PCIe link establishment due to breaking 20mS time limit of PRSTn.

Change-Id: I11693927a4f56d8438533e3a004758baa83bea18
---

diff --git a/arch/gaudi2/arc/Makefile b/arch/gaudi2/arc/Makefile
index 74d9197..e63d810 100644
--- a/arch/gaudi2/arc/Makefile
+++ b/arch/gaudi2/arc/Makefile
@@ -9,4 +9,5 @@
 PREBOOT_SRCS_C += $(shell ls $(PREBOOT_ROOT_DIR)/lib/preboot/thermal_v2.c)
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV_ARCH)/security.c)
 
+image_validation: image_validation_common
 image_validation: image_validation_preboot
